src/core/lib/slice/slice_utils.h

```diff
@@ -31,170 +31,13 @@
 
 namespace grpc_core {
 extern uint32_t g_hash_seed;
```

Comment:
Should this move to slice_internal.h?

Actually, there's so little left in this file, maybe we should just move all of it to slice_internal.h?

Reply:
yeah ok - we had needed the split to make static metadata layer properly in the build, but that requirement went so it is time.